mobile/lib/features/trade/position_list_item.dart

```diff
@@ -123,9 +124,11 @@ class PositionListItem extends StatelessWidget {
                 width: 10,
               ),
               ElevatedButton(
-                onPressed: () {
-                  // TODO: bottom sheet to close position
-                },
+                onPressed: notNullPosition.positionState == PositionState.closing
+                    ? null
+                    : () async {
+                        await onClose();
+                      },
```

Comment:
I tested it again and it does work, but it might not happen immediately.
To make this better we would have to e.g. track the state in widget using widget state independent of the state of the position. Upon pressing the button we would set it to disabled internally - this could be released by e.g. a timer. The position's state would disable the button independently. 

I did not add this logic yet.